app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by(email: params[:email])
+    if user && !user.activated? && user.authenticated?(:activation, params[:id])
+      user.activate
+      log_in user
+      flash[:success] = t ".success_message"
+      redirect_to user
+    else
+      flash[:danger] = t ".failure_message"
+      redirect_to root_url
```

Comment:
```suggestion
      redirect_to root_path
```
trong các controller thì dùng path thôi e